README.md

```diff
@@ -181,67 +58,38 @@ Now the repository is ready for code changes.
 
 The `package.json` also contains scripts (runnable by calling `yarn <script-name>` or `npm run <script-name>`):
 
-- `lint` - lint the code (use `lint --fix` to run autofix)
-- `format` - check the code formatting (use `firmat:fix` to autoformat)
-- `prepush` - run `test`, `lint` and `format` checks. This should run without errors before you push anything to git.
-
-Lastly, to build a local artifact run `yarn build` or `npm run build`.
-
-**Note**: The project needs to be built (into the `dist` folder) to run cli commands.
-
-**Note**: You can change url of API requests by setting `SUPERFACE_API_URL` environment variable to desired base url.
+- `lint` - lint the code (use `lint:fix` to run autofix)
+- `format` - check the code formatting (use `format:fix` to autoformat)
+- `prepush` - run `lint` and `format` checks. This should run without errors before you push anything to git.
 
 ### Adding new capability
 
-First, create new profile:
-
-```
- station create profile {scope}/{usecase}
-```
-
-Edit created .supr file:
-
-Secondly, create new provider:
+#### Create new profile
 
-```
- station create provider {provider}
+```shell
+yarn superface create --profileId [scope](optional)/[name] --profile --profileFileName capabilities/[scope]/[name]/profile.supr
```

Comment:
Why is the superface call prepended with `yarn`?

Reply:
So it uses the one within project and not globally installed.